web/themes/custom/cybersecurity/cybersecurity.libraries.yml

```diff
@@ -11,3 +11,5 @@ global-js:
   dependencies:
     - core/jquery
     - core/jquery.once
+    - core/drupal
```

Comment:
Should be able to get away with not requiring /drupal or /drupalSettings if you don't need the drupal object.